packages/gatsby/src/commands/graphiql.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+const express = require(`express`)
+const graphqlHTTP = require(`express-graphql`)
+const { store } = require(`../redux`)
+const bootstrap = require(`../bootstrap`)
+
+module.exports = async (program: any) => {
+  let { port, host } = program
+  port = typeof port === `string` ? parseInt(port, 10) : port
+
+  // bootstrap to ensure schema is in the store
+  await bootstrap(program)
```

Comment:
I wonder if we should split bootstrap into a few pieces so we can just generate the schema and data here without running the other code

Reply:
👍 to this. But maybe in a separate PR?